CMakeLists.txt

```
@@ -53,9 +53,9 @@ if(APPLE)
   set(QT_MODULES ${QT_MODULES} DBus)
 endif()
 
-find_package(Qt5 5.15 COMPONENTS ${QT_MODULES} LinguistTools REQUIRED)
+find_package(Qt5 5.12 COMPONENTS ${QT_MODULES} LinguistTools REQUIRED)
```

```suggestion
if (QtVersion STREQUAL "5.12")
    find_package(Qt5 5.12 COMPONENTS ${QT_MODULES} LinguistTools REQUIRED)
else()
    find_package(Qt5 5.15 COMPONENTS ${QT_MODULES} LinguistTools REQUIRED)
endif()
```

I'm using Arch and there 5.15 is standard, so I think it would be the best to introduce a new variable to easily change the Qt Version

---

find_package(Qt5 5.12 ...) defines the minimum required Qt Version to be 5.12 - all later versions (e.g. 5.15) will work fine. I think the if - elseif - else - endif is not required.

The required flatpak version to be 5.15 works for me.

---

Ah ok then it is fine